pyrefinebio/computed_file.py

```diff
@@ -85,37 +85,40 @@ def search(cls, **kwargs):
 
         returns: list of ComputedFile
 
-        parameters:
+        valid filters:
 
-            id (int):
+            id (int): filter based on the id of the computed file
 
-            samples (str):
+            samples (str): filter based on the accession code for samples related to the computed file
 
-            is_qn_target (bool):
+            is_qn_target (bool): filter based on if the computed file is a qn target
 
-            is_smashable (bool):
+            is_smashable (bool): filter based on if the computed file is smashable
```

Comment:
Mmm, I don't love that the word `smashable` is user facing. Probably never should have made it into the code, but maybe we can at least change this to say `filter based on if the computed file is able to added to a normalized Dataset"